### PR TITLE
Feature/hosted script

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

File: scripts/publish-github-release.sh [hunk 1]
```diff
@@ -73,25 +73,25 @@ else
 fi
 
 # 4. Copy binaries from :sdk
```
Comment (jaelim-ms) [inline scripts/publish-github-release.sh]: We can probably improve this step later.
I think we don't need to move this to that specific directory anymore, there is a copy task in VSTS.

File: scripts/publish-storage-account.sh [hunk 2]
```diff
@@ -11,14 +13,16 @@ FILES="$(find sdk -name $BINARY_FILE_FILTER)"
...
-cd $BITRISE_DEPLOY_DIR
+cd $BUILD_ARTIFACTSTAGINGDIRECTORY
 mkdir $ARCHIVE
 cp $BINARY_FILE_FILTER $ARCHIVE
 zip -r $ZIP_FILE $ARCHIVE
+cd -
 
 # Upload file
-$1azure storage blob upload $ZIP_FILE sdk
+AZURE_STORAGE_ACCESS_KEY=$AZURE_STORAGE_ACCESS_KEY \
```
Comment (jaelim-ms) [inline scripts/publish-storage-account.sh]: What is this line?

File: scripts/run-code-coverage.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# We can't run emulator as a daemon
```
Comment (jaelim-ms) [inline scripts/run-code-coverage.sh]: It is really bad that VSTS doesn't have an option for emulator. It was in Android Build task but it is deprecated. 😞

File: scripts/publish-storage-account.sh (hunk 2 above)
Comment (guperrot) [reply scripts/publish-storage-account.sh]: expose environment variable just during that call

File: scripts/publish-github-release.sh (hunk 1 above)
Comment (guperrot) [reply scripts/publish-github-release.sh]: Yeah it actually works already without it, but this is nice to have when running locally

File: scripts/publish-storage-account.sh (hunk 2 above)
Comment (jaelim-ms) [reply scripts/publish-storage-account.sh]: Oh.. OK. Probably I need this in Apple SDK build. Currently it exports the environment variables.

**Pull request merged**